### PR TITLE
Shortcut keys for 'View Path Issues' and 'Cutaway View' view modes

### PR DESCRIPTION
**Pull request by xp4xbox**
#7332

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Never update any language besides `en-GB` in this repository. All other languages are maintained in Localisation and any changes here are overwritten automatically.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 0.1.3 (in development)
 ------------------------------------------------------------------------
+- Feature: [#7332] Keyboard shortcuts for view path issues and cutaway view
```
Comment (janisozaur) [inline distribution/changelog.txt]: they go in order

File: src/openrct2/localisation/StringIds.h [hunk 2]
```diff
@@ -1956,6 +1956,8 @@ enum {
     STR_SHORTCUT_SHOW_MAP = 2523,
     STR_SHORTCUT_SCREENSHOT = 2524,
     STR_SHORTCUT_KEY_UNKNOWN = 2525,
+    STR_SHORTCUT_VIEW_CLIPPING = 6005,
+    STR_SHORTCUT_HIGHLIGHT_PATH_ISSUES_TOGGLE = 6233,
```
Comment (AaronVanGeffen) [inline src/openrct2/localisation/StringIds.h]: These should be added at the end of the file.

**Comment by Gymnasiast** [comment]
Do not merge yet.

File: src/openrct2-ui/input/KeyboardShortcuts.cpp [hunk 3]
```diff
@@ -305,4 +305,6 @@ const uint16 KeyboardShortcuts::DefaultKeys[SHORTCUT_COUNT] =
     PLATFORM_MODIFIER | SDL_SCANCODE_L,         // SHORTCUT_LOAD_GAME
     SDL_SCANCODE_B,                             // SHORTCUT_CLEAR_SCENERY
     SDL_SCANCODE_7,                             // SHORTCUT_GRIDLINES_DISPLAY_TOGGLE
+    SDL_SCANCODE_U,                             // SHORTCUT_VIEW_CLIPPING
```
Comment (Gymnasiast) [inline src/openrct2-ui/input/KeyboardShortcuts.cpp]: I don't think this should have a shortcut key by default. It's a tool for more advanced users and not for casual gameplay.

**Pull request merged**